drivers/storage/portworx/component/portworx_api.go

```diff
@@ -264,6 +265,15 @@ func getPortworxAPIDaemonSetSpec(
 		},
 	}
 
+	// If CSI is enabled then run the csi-node-driver-registrar pods in the same daemonset
+	if pxutil.IsCSIEnabled(cluster) {
```

Comment:
Let's add the px version 2.13 check here too. We can avoid creating duplicate registrar containers that way.